src/httpserver.cpp

```diff
@@ -118,7 +112,7 @@ class WorkQueue
         if (queue.size() >= maxDepth) {
             return false;
         }
-        queue.push_back(item);
+        queue.emplace_back(std::unique_ptr<WorkItem>(item));
```

Comment:
Creating the unique_ptr here defeats the purpose of the emplace :)
Passing it item directly would mean that the unique_ptr would be constructed in-place. As is, it's moved instead.

Not worth worrying about, just pointing it out.